### PR TITLE
Update 0x11-V2-Authentication.md

### PR DESCRIPTION
**Pull request by csfreak92**
Updating 2.3.1 to cover the use case of admins changing user's passwords.

This Pull Request relates to issue #1239.

### CODE REVIEW

**Comment by tghosth** [comment]
Requested modification to the PR in issue #1239

**Comment by csfreak92** [comment]
@tghosth, just updated the PR. I think section 2.3.x makes sense for this new requirement. For the CWE I had researched a bit for it, but the closest one I can see aligning to it is: CWE 620 [(https://cwe.mitre.org/data/definitions/620.html)]. I may need to dig deeper for the NIST equivalence, but maybe we can merge this first if you are satisfied with this PR and I will follow up with the NIST mapping if ever?

File: 5.0/en/0x11-V2-Authentication.md [hunk 1]
```diff
@@ -75,9 +75,11 @@ Note: Passwords are not to have a maximum lifetime or be subject to password rot
...
+| **2.3.1** | [MODIFIED, MERGED FROM 2.5.1] Verify system generated initial passwords or activation codes are securely randomly generated, at least 6 characters long, may contain letters and numbers, expire after a short period of time, and are single-use. These initial secrets must not be permitted to become the long term password.  | ✓ | ✓ | ✓ | 330 | 5.1.1.2 / A.3 |
 | **2.3.2** | Verify that enrollment and use of user-provided authentication devices are supported, such as a U2F or FIDO tokens. | | ✓ | ✓ | 308 | 6.1.3 |
 | **2.3.3** | Verify that renewal instructions are sent with sufficient time to renew time bound authenticators. | | ✓ | ✓ | 287 | 6.1.4 |
+| **2.3.4** | [ADDED] System administrators should not be able to change or choose any user's password, but rather only be able to initiate the password reset process for the user. | ✓ | ✓ | ✓ | 620 |  |
```
Comment (tghosth) [inline 5.0/en/0x11-V2-Authentication.md:81]: Please confirm that you are comfortable with the subtle change I made here, see: https://github.com/OWASP/ASVS/pull/1263/commits/eed125c02c8aabd64ac634b9f5166e994383ce21

**Review by tghosth — COMMENTED** [review]
Created one comment

File: 5.0/en/0x11-V2-Authentication.md (hunk 1 above)
Comment (tghosth) [reply 5.0/en/0x11-V2-Authentication.md:81]: @csfreak92

Comment (csfreak92) [reply 5.0/en/0x11-V2-Authentication.md:81]: This sounds good to me.

**Review by tghosth — APPROVED** [review]

**Pull request merged**